packages/web_benchmarks/lib/client.dart

```diff
@@ -4,11 +4,14 @@
 
 import 'dart:async';
 import 'dart:convert' show json;
-import 'dart:html' as html;
+import 'dart:js_interop';
 import 'dart:math' as math;
 
+import 'package:web/helpers.dart' as html;
```

Comment:
nit can we drop to html prefix? seems misleading since this isn't dart:html

Reply:
done!